src/style.css

```diff
@@ -0,0 +1,28 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 0;
+  padding-bottom: 14px;
+  margin: 2px 2px 0;
```

Comment:
You can only add margin-right: 4px like in mockup and remove margin for the last element in a row